service/controller/resource/tccp/create_test.go

```diff
@@ -45,20 +48,26 @@ func Test_Controller_Resource_TCCP_Template_Render(t *testing.T) {
 			name:           "case 0: basic test, route53 enabled",
 			cr:             unittest.DefaultCluster(),
 			ctx:            unittest.DefaultContext(),
+			cpAzs:          []string{"eu-central-1a"},
+			cpReplicas:     1,
 			errorMatcher:   nil,
 			route53Enabled: true,
 		},
 		{
 			name:           "case 1: basic test, route53 disabled",
 			cr:             unittest.DefaultCluster(),
 			ctx:            unittest.DefaultContext(),
+			cpAzs:          []string{"eu-central-1a"},
+			cpReplicas:     1,
 			errorMatcher:   nil,
 			route53Enabled: false,
 		},
 		{
-			name: "case 2: basic test with api whitelist enabled",
-			cr:   unittest.DefaultCluster(),
-			ctx:  unittest.DefaultContext(),
+			name:       "case 2: basic test with api whitelist enabled",
+			cr:         unittest.DefaultCluster(),
+			ctx:        unittest.DefaultContext(),
+			cpAzs:      []string{"eu-central-1a"},
+			cpReplicas: 1,
```

Comment:
The test input is always the same. Would it make sense to consider some variation here?

Reply:
I wonder if it make sense to have 3  replicas as that value si anyway never used. maybe i can randomly assign Azs